content/applications/finance/accounting/payments/follow_up.rst

```diff
@@ -14,44 +14,61 @@ Configuration
 =============
 
 To configure a :guilabel:`Follow-Up Action`, go to :menuselection:`Accounting --> Configuration -->
-Follow-up Levels`. Several follow-up actions are available by default, and you can customize them
-any way you want or create new ones to trigger the following actions:
+Follow-up Levels`. Several follow-up actions are available by default under the
+:guilabel:`Notification`, but you can create new ones if required, and can change the **name** and
+the **number of days** after which the follow-up actions are triggered. The follow-up
+:guilabel:`Actions` available are:
 
-- Send an email;
-- :ref:`Send an SMS message <pricing/pricing_and_faq>`;
-- Print a letter;
-- :ref:`Send a letter <customer_invoices/snailmail>`;
-- Manual action (creates a task).
+- :guilabel:`Send Email`;
```

Comment:
Send an Email 
or 
Send Emails 
but you always used singular

Reply:
It's written "Send Email" word-for-word in Odoo, hence why I used a :guilabel: